app/Http/Controllers/Api/V1/TeamController.php

```diff
@@ -625,7 +645,11 @@ public function addMembersToProject(TeamAddMemberRequest $addMemberRequest, Team
      * @param Project $project
      * @return Response
      */
-    public function removeMemberFromProject(TeamRemoveMemberRequest $removeMemberRequest, Team $team, Project $project)
+    public function removeMemberFromProject(
```

Comment:
Please revert it back to previous, if it makes horizontal scroll then only we use break line